### PR TITLE
Migrate to core20

### PR DESCRIPTION
**Pull request by khiemdoan**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -37,8 +30,8 @@ parts:
     prime:
       - jetbrains-studio.desktop
   android-studio:
-    after: [gnome]
-    plugin: nil
+    plugin: dump
```
Comment (om26er) [inline snap/snapcraft.yaml:33]: please change that back to `nil`

Comment (khiemdoan) [reply snap/snapcraft.yaml:33]: We need to use `dump` to access the `get_latest.py` file.

Comment (om26er) [reply snap/snapcraft.yaml:33]: the `source: .` stanza will help with that. We dont need to use dump.

Comment (khiemdoan) [reply snap/snapcraft.yaml]: I have changed that :smile: :smile:

**Comment by om26er** [comment]
@popey I think this PR is good to land now.

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
-script:
-  - export PATH=/snap/bin:$PATH
-  - sudo snapcraft cleanbuild
-  - sudo cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - sudo snapcraft --use-lxd
 after_success:
-  - sudo snap install transfer
-  - timeout 180 sudo /snap/bin/transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - timeout 180 transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
```
Comment (om26er) [inline .travis.yml:26]: btw, we could get rid of this line as well. The underlying service will go offline on 30th October, so may just get rid of it now.

Comment (khiemdoan) [reply .travis.yml:26]: I will do it in the next PR.

**Pull request merged**